src/frontend/src/flows/recovery/inputSeedPhrase.ts

```diff
@@ -48,18 +51,33 @@ const init = (userNumber: bigint): Promise<string | null> =>
     const inputSeedPhraseInput = document.getElementById(
       "inputSeedPhrase"
     ) as HTMLInputElement;
-    const warningBadAnchor = document.getElementById(
-      "warningBadAnchor"
-    ) as HTMLParagraphElement;
+
+    // Look up the warningsDiv element early; this must not be done inside in the warnings
+    // generation callback since -- due to debouncing -- it may be called after some period of
+    // time and the user may have left the page; we may effectively pick up the wrong element.
+    const warningsDiv = document.getElementById("warnings") as HTMLDivElement;
+
+    // Debounce the warning generation as not to spam the user
```

Comment:
Ah, didn't know! Doesn't that mean that the user may input the phrase, then click `Continue` without ever seeing the warnings? I'm happy to disable the button until the warnings have been shown but that feels a bit heavy

Reply:
We disable the button yes, and when they press enter or click outside the input, the errors appear or the buttons becomes enabled.

Since, you want to allow the user to continue even with warnings, it might not be the best approach.

I don't know, we always rely on Mischa for UX decisions.

Reply:
ok, talked to Mischa; first of all he said it didn't really matter, since this should happen pretty rarely. He suggested we instead only show warnings when clicking "continue"; I realized this makes things much more complicated since then there's the question of what happens when "continue" is clicked the second time: do we submit, or do we recompute the warnings?

Then after talking to David we agreed that the fact that the whole box jitters is quite annoying; also agreed that `onblur` wouldn't really work because there's the risk of directly clicking "continue" without seeing the warnings (because they're not generated unless you click out of the text area). I've fixed things up a bit, will update the screen recording in the PR description. Thanks!